static/app/actionCreators/serverSideSampling.tsx

```diff
@@ -14,12 +14,13 @@ export function fetchSamplingSdkVersions({
   orgSlug: Organization['slug'];
   projectID: Project['id'];
 }): Promise<SamplingSdkVersion[]> {
-  const {samplingDistribution} = ServerSideSamplingStore.getState();
-  const {startTimestamp, endTimestamp, project_breakdown} = samplingDistribution;
+  const {distribution} = ServerSideSamplingStore.getState();
+  const {startTimestamp, endTimestamp, project_breakdown} = distribution.data ?? {};
+
+  ServerSideSamplingStore.fetchSdkVersions();
```

Comment:
This is a nit but the name of all these `fetch*()` functions on the store seems to imply that they'll do the fetching but in reality, they just mark the start of the fetch and then you have to explicitly fetch and load the data into the store. Would be nice if they had clearer names.

Reply:
sure thing... will add a start fetching or something 😉